### DISCUSSION TITLE
`repo/ìssues/new` url params are not supported by the app

### DISCUSSION THREAD

**Discussion by leonhma** [community/community#32208] [Mobile] [answered]
I just learned that you can control issue forms via url params, so I made my app generate a link that leads users to a prefilled bug report when encountering an error. This works fine if I just open the link on my desktop, but if I scan the qr-code to open the link on my phone, the app opens and I get shown the 'Choose Template' page.

The url for context is `https://github.com/leonhma/obsidian-functionplot/issues/new?labels=bug&template=BUG_REPORT.yml&title=Unhandled%20Error&what-happened=&checkboxes=false%2Cfalse&ideas=&additional=%23%23%23%20Log%0A%0A%60%60%60text%0A%60%60%60%0A%0A`.
As you can see, it already specifies things like the issue type, label and prefilled fields, but all that is lost when you open the link via the github app (the default url handler). A possible workaround is use a redirect (link to another url that redirects to this url), but that would either be abusing an open redirect or need another server.

The app currently only opens the issue form on github (web) anyways, so please make it also forward the url params.

tested on Android 13

**Comment by mxie** [comment, marked as answer]
Thanks for reporting this @leonhma! This does seem like a bug on our end as I was able to reproduce this. I'll forward this over to our team to have a look.

**Reply by mxie** [reply]
Hi @leonhma! I just wanted to let you know that we just landed a fix in our beta build today. If you're on the production build, you'll see it fixed next week instead. Please let us know if you're still having issues though. Thanks!

**Comment by 16275009713915100403** [comment]
[[]()]()

**Comment by mufa988** [comment]
https://github.com/orgs/community/discussions/20214

**Comment by mufa988** [comment]
> I just learned that you can control issue forms via url params, so I made my app generate a link that leads users to a prefilled bug report when encountering an error. This works fine if I just open the link on my desktop, but if I scan the qr-code to open the link on my phone, the app opens and I get shown the 'Choose Template' page.
> 
> 
> 
> The url for context is `https://github.com/leonhma/obsidian-functionplot/issues/new?labels=bug&template=BUG_REPORT.yml&title=Unhandled%20Error&what-happened=&checkboxes=false%2Cfalse&ideas=&additional=%23%23%23%20Log%0A%0A%60%60%60text%0A%60%60%60%0A%0A`.
> 
> As you can see, it already specifies things like the issue type, label and prefilled fields, but all that is lost when you open the link via the github app (the default url handler). A possible workaround is use a redirect (link to another url that redirects to this url), but that would either be abusing an open redirect or need another server.
> 
> 
> 
> The app currently only opens the issue form on github (web) anyways, so please make it also forward the url params.
> 
> 
> 
> tested on Android 13

**Comment by wdpmm515188** [comment]
这个版本主页上面的搜索不能用，荣耀10青春版、鸿蒙2.0。
